docs/ubuntu-installation-guide.md

```diff
@@ -5,14 +5,14 @@ Note:
 If you are installing on a system that already has Clear Containers 2.x
 installed, first read [the upgrading document](upgrading.md).
 
-Clear Containers **3.0** is available for Ubuntu\* **16.04**.
+Clear Containers **3.0** is available for Ubuntu\* **16.04** , **16.10** and **17.04**.
 
 This step is only required in case Docker is not installed on the system.
 1. Install the latest version of Docker with the following commands:
 
 ```
-$ sudo -E apt-get -y install apt-transport-https ca-certificates curl software-properties-common
-$ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+$ sudo -E apt-get -y install apt-transport-https ca-certificates wget software-properties-common
+$ wget -qO - https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```

Comment:
Why are we switching to `wget` by the way?

Reply:
@jodh-intel there is an issue with curl with ubuntu 17.04 
$ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
curl: (35) Unknown SSL protocol error in connection to ...
gpg: no valid OpenPGP data found.

Reply:
It's working for me on a 17.04 box using this version of `curl`:

```
$ lsb_release -r                                                                
Release:        17.04
$ curl -fsSL https://download.docker.com/linux/ubuntu/gpg &>/dev/null && echo ok
ok
$ dpkg -l curl|grep ^ii|awk '{print $3}'                                        
7.52.1-4ubuntu1.2
```

Is you system fully up-to-date?

Reply:
@jodh-intel, yes
$ lsb_release -r
Release:        17.04
$ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
curl: (35) Unknown SSL protocol error in connection to proxy-chain.intel.com:911
gpg: no valid OpenPGP data found.
$ dpkg -l curl|grep ^ii|awk '{print $3}'
7.52.1-4ubuntu1.2

Reply:
ok - looks like a proxy issue. `wget` it is then! ;)

Reply:
@jodh-intel :)